master/internal/command/authz_tensorboard_iface.go

```diff
@@ -0,0 +1,36 @@
+package command
+
+import (
+	"context"
+
+	"github.com/determined-ai/determined/proto/pkg/tensorboardv1"
+
+	"github.com/determined-ai/determined/master/internal/authz"
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+// TensorboardAuthZ describes authz methods for Tensorboards.
+type TensorboardAuthZ interface {
+	// GET /api/v1/tensorboards/:tb_id
+	CanGetTensorboard(
+		ctx context.Context, curUser *model.User, tb *tensorboardv1.Tensorboard,
+	) (canGetTensorboard bool, serverError error)
+
+	// GET /api/v1/tensorboards
+	FilterTensorboards(
```

Comment:
we only really need the workspace ids of the tensorboards here. what you have here feels natural but how do we keep it consistent with the NCSAuthZ iface if we wanted to? 
 https://github.com/determined-ai/determined/pull/5609/files#diff-5306290d2cf5fbcbbf54ff35798d9a111e710c775399f4f09c996079fb8f98c8R27